step-controller/ControllerServer.run.xml

```diff
@@ -3,7 +3,7 @@
     <option name="MAIN_CLASS_NAME" value="step.controller.ControllerServer" />
     <module name="step-controller-backend" />
     <option name="PROGRAM_PARAMETERS" value="-config=&quot;src/test/resources/step.properties&quot;" />
-    <option name="VM_PARAMETERS" value="-Dhttp.keepAlive=true -Dhttp.maxConnections=50 -Dlogback.configurationFile=$PROJECT_DIR$/../step-distribution/step-distribution-parent/step-distribution-controller/template-controller/bin/logback.xml" />
+    <option name="VM_PARAMETERS" value="-Dhttp.keepAlive=true -Dhttp.maxConnections=50 -Dlogback.configurationFile=C:\projects\step-distribution\step-distribution-parent\step-distribution-controller\template-controller\bin\logback.xml" />
```

Comment:
this change should not be committed, we cannot have an absolute path here as it won't be the same for everyone

Reply:
Thanks. Yes, I know. I was commited by mistake